strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/cron/jobs/RebuildMavenMetadataCronJobTestIT.java

```diff
@@ -222,21 +209,21 @@ public void testRebuildMetadataInRepository()
                     Versioning versioning1 = metadata1.getVersioning();
                     Versioning versioning2 = metadata1.getVersioning();
 
-                    assertEquals("Incorrect artifactId!", artifact1.getArtifactId(), metadata1.getArtifactId());
-                    assertEquals("Incorrect groupId!", artifact1.getGroupId(), metadata1.getGroupId());
+                    assertEquals(artifact1.getArtifactId(), metadata1.getArtifactId(), "Incorrect artifactId!");
+                    assertEquals(artifact1.getGroupId(), metadata1.getGroupId(), "Incorrect groupId!");
 
-                    assertEquals("Incorrect artifactId!", artifact2.getArtifactId(), metadata2.getArtifactId());
-                    assertEquals("Incorrect groupId!", artifact2.getGroupId(), metadata2.getGroupId());
+                    assertEquals(artifact2.getArtifactId(), metadata2.getArtifactId(), "Incorrect artifactId!");
+                    assertEquals(artifact2.getGroupId(), metadata2.getGroupId(), "Incorrect groupId!");
 
-                    assertNotNull("No versioning information could be found in the metadata!", versioning1.getVersions()
-                                                                                                          .size());
-                    assertEquals("Incorrect number of versions stored in metadata!", 1, versioning1.getVersions()
-                                                                                                   .size());
+                    assertNotNull(versioning1.getVersions()
+                            , "No versioning information could be found in the metadata!");
+                    assertEquals(1, versioning1.getVersions()
+                                               .size(), "Incorrect number of versions stored in metadata!");
 
-                    assertNotNull("No versioning information could be found in the metadata!", versioning2.getVersions()
-                                                                                                          .size());
-                    assertEquals("Incorrect number of versions stored in metadata!", 1, versioning2.getVersions()
-                                                                                                   .size());
+                    assertNotNull(versioning2.getVersions()
+                            , "No versioning information could be found in the metadata!");
```

Comment:
Cosmetic: Could you please re-indent this as per the coding convention? Thanks! :)